### PR TITLE
chore: bump all versions

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**

**Did you add tests for your changes?**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

**Review by anshumanv — DISMISSED** [review]

**Review by evilebottnawi — COMMENTED** [review]
Who is bump version in `dependencies`?

File: packages/generators/package.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
     "access": "public"
   },
   "dependencies": {
-    "@webpack-cli/package-utils": "^1.0.1-alpha.4",
+    "@webpack-cli/package-utils": "^1.0.1-rc.0",
     "@webpack-cli/utils": "^1.0.1-alpha.5",
```
Comment (evilebottnawi) [inline packages/generators/package.json:14]: Missing

Comment (evenstensberg) [reply packages/generators/package.json:14]: Missing what

Comment (evenstensberg) [reply packages/generators/package.json:14]: nvm

**Review by evilebottnawi — APPROVED** [review]

**Comment by evilebottnawi** [comment]
Do we need something in changelog?

**Review by anshumanv — APPROVED** [review]

**Comment by evenstensberg** [comment]
Haven't been released yet, I think we need to change the names first then bump the versions with lerna publish (theory), might be that we need to keep the alpha versions as lerna might complain, I'll inform you at Slack

**Pull request merged**